tests/helpers/strings.spec.js

```diff
@@ -51,4 +51,66 @@ describe('strings', () => {
         expect(strings.capitalizeEach(1)).toEqual(1);
     });
 
+    /* concat */
+    it('concat should return the param value if only one parameter(string) is provided', () => {
+        expect(strings.concat('hello')).toEqual('hello');
+    });
+
+    it('concat should return the param value if only one parameter(integer) is provided', () => {
+        expect(strings.concat(5)).toEqual('5');
+    });
+
+    it('concat should return concatenation of all param values(string)', () => {
+        expect(strings.concat('hello', ' ', 'world', '!!!')).toEqual('hello world!!!');
+    });
+
+    it('concat should return concatenation of all param values(string and integer)', () => {
+        expect(strings.concat('I have got', ' ', 4, ' ', 'apples.')).toEqual('I have got 4 apples.');
+    });
+
+    it('concat should return concatenation of all param values(integer)', () => {
+        expect(strings.concat(1, 2, 3, 4)).toEqual('1234');
+    });
+
+    it('concat should return empty string if no params provided', () => {
+        expect(strings.concat()).toEqual('');
+    });
+
+    it('concat should return concatenation of boolean value if boolean params provided', () => {
+        expect(strings.concat(true, false, true, false)).toEqual('truefalsetruefalse');
+    });
+
+    it('concat should return concatenation of boolean value if null params provided', () => {
+        expect(strings.concat(null, 'abc')).toEqual('nullabc');
+    });
+
+    /* join */
+    it('join should join the values of array using the delimeter provided', () => {
+        expect(strings.join(['Hands', 'legs', 'feet'], ' & ')).toEqual('Hands & legs & feet');
+    });
+
+    it('join should return the first value of array if size of the array is equals to 1', () => {
+        expect(strings.join(['Hands'], ' & ')).toEqual('Hands');
+    });
+
+    it('join should return empty string if size of array is zero', () => {
+        expect(strings.join([], ' & ')).toEqual('');
+    });
+
+    it('join should return empty string if first parameter is null', () => {
+        expect(strings.join(null, ' & ')).toEqual('');
+    });
+
+    it('join should return empty string if both the array and delimeter is null', () => {
+        expect(strings.join(null, null)).toEqual('');
+    });
+
+    it('join should return concatenation of elements of array using null if the delimeter is null', () => {
+        expect(strings.join(['Hands', 'legs', 'feet'], null)).toEqual('Handsnulllegsnullfeet');
+    });
+
+    it('join should return concatenation of elements of array using false if the delimeter is false', () => {
+        expect(strings.join(['Hands', 'legs', 'feet'], false)).toEqual('Handsfalselegsfalsefeet');
+    });
```

Comment:
You are just testing the helper function. Also add few tests with compiled helper. 

For eg: write few tests like this: 

```javascript
    import Handebars from 'handlebars';
    ...

    it('sprintf should work as expected after compilation (Basic support)', () => {
        var template = Handlebars.compile("{{join items ' ' }}");
        var items = ['Hello', 'World'];

        expect(template({items})).toEqual('Hello World');
    });
```